Markdown/Markdown.sublime-syntax

```diff
@@ -2099,12 +2182,12 @@ contexts:
       captures:
         1: markup.italic.markdown punctuation.definition.italic.end.markdown
         2: punctuation.definition.bold.end.markdown
-      pop: true
+      pop: 1
     - match: _\b
-      scope: punctuation.definition.italic.end.markdown
+      scope: markup.italic.markdown punctuation.definition.italic.end.markdown
```

Comment:
missing test

Reply:
Not sure what you mean. This and related changes are required to maintain behavior. No tests missing. Just revert this change and learn how existing tests are failing.